### PR TITLE
Fix getNonStandardShortcuts

### LINKED ISSUE #3922: [BUG] Shortcut: some keys create a warning for preg_match()
**Describe the bug**
some keys for the shortcuts create a warning, f.e. `?` or `/` or `\`

grafik

**To Reproduce**
1. run your application in development mode (data/config.php => L3: `'environment' => 'development',`
2. Go to 'shortcuts config page'
2. edit the shortcuts to `?` or `/` or `\`
3. See warning

**Environment information (please complete the following information):**
- FreshRSS version: 1.18.1

### PR DESCRIPTION
**Pull request by Alkarex**
#fix https://github.com/FreshRSS/FreshRSS/issues/3922
Hot fix. The `getNonStandardShortcuts()` could maybe be re-written to work with array elements.

### CODE REVIEW

**Comment by math-GH** [comment]
works fine

File: lib/lib_rss.php [hunk 1]
```diff
@@ -538,10 +538,8 @@ function getNonStandardShortcuts($shortcuts) {
 	$standard = strtolower(implode(' ', SHORTCUT_KEYS));
 
 	$nonStandard = array_filter($shortcuts, function ($shortcut) use ($standard) {
-		if (false !== strpos($shortcut, ' ')) {
-			return true;
-		}
-		return !preg_match("/${shortcut}/i", $standard);
+		$shortcut = trim($shortcut);
+		return $shortcut !== '' & stripos($standard, $shortcut) === false;
```
Comment (Frenzie) [inline lib/lib_rss.php:542]: Do we have both a strpos and a stripos or is one of those a typo?

Comment (math-GH) [reply lib/lib_rss.php:542]: `stripos` is case-insensitiv (https://www.php.net/manual/en/function.stripos.php). `strpos` is case-sensitive

Comment (Alkarex) [reply lib/lib_rss.php:542]: No error @Frenzie , but it would be nicer with a rewrite using array elements (but that is less urgent)

**Comment by Alkarex** [comment]
Let's merge this hot fix and potentially come back to a better implementation if anyone is motivated :-)

**Pull request merged**